pkgs/development/compilers/gcc-ng/11/gcc/default.nix

```diff
@@ -0,0 +1,207 @@
+{ lib, stdenv, targetPackages, fetchurl, fetchpatch
+, gcc_tools_meta, gcc_src, version
+
+, langAda ? false
+, langC ? true
+, langCC ? true
+, langD ? false
+, langFortran ? false
+, langGo ? false
+, langJava ? false
+, langObjC ? stdenv.targetPlatform.isDarwin
+, langObjCpp ? stdenv.targetPlatform.isDarwin
+
+, reproducibleBuild ? true
+, profiledCompiler ? false
+, langJit ? false
+, texinfo ? null
+, perl ? null # lib.optional, for texi2pod (then pod2man)
+, gmp, mpfr, libmpc, gettext, which, patchelf
+, libelf                      # lib.optional, for link-time optimizations (LTO)
+, isl ? null
+, zlib ? null
+, name ? "gcc"
+, gnused ? null
+, nonRelease ? false, flex ? null, bison ? null # Convenience for working on GCC
```

Comment:
Please stop adding new occurrences of ``? null``. If the feature flag is deactivated the package is no evaluated and does not need to be overwritten to null.

Reply:
This derivation may be called from a bootstrap scope where these libraries are missing.